includes/class-sensei-analysis-overview-list-table.php

```diff
@@ -804,6 +811,28 @@ public function search_button( $text = '' ) {
 		return $text;
 	}
 
+	/**
+	 * Add the sum of days taken by each student to complete a lesson with returning lesson row.
+	 *
+	 * @param array $clauses Incoming clauses.
```

Comment:
Could we make this a bit more descriptive? "Incoming clauses" doesn't tell me too much. We could borrow directly from the [hook docs](https://developer.wordpress.org/reference/hooks/posts_clauses/) - "Associative array of the clauses for the query".

Reply:
Yeah absolutely, thanks for the suggestion. Updated it accordingly here https://github.com/Automattic/sensei/pull/4883/commits/924df553acadf51176832236234004ca471760bf